tsl/src/fdw/scan_plan.c

```diff
@@ -942,6 +1117,415 @@ add_foreign_grouping_paths(PlannerInfo *root, RelOptInfo *input_rel, RelOptInfo
 		fdw_add_upper_paths_with_pathkeys_for_rel(root, grouped_rel, NULL, create_path);
 }
 
+/*
+ * add_foreign_ordered_paths
+ *		Add foreign paths for performing the final sort remotely.
+ *
+ * Given input_rel contains the source-data Paths.  The paths are added to the
+ * given ordered_rel.
+ */
+static void
+add_foreign_ordered_paths(PlannerInfo *root, RelOptInfo *input_rel, RelOptInfo *ordered_rel,
```

Comment:
This whole function is never called according to code coverage, so we should not add it (now).

Reply:
@erimatnor this function and others will be called in my follow on patch for upper rel handling that I am stabilizing currently